examples/apps/colorapp/ecs/ecs-colorapp.yaml

```diff
@@ -32,6 +32,13 @@ Parameters:
     Type: String
     Description: Task definition for ColorTeller Black Service
 
+  Path:
```

Comment:
Could we make this parameter more specific? How about `LoadBalancerPath`?

Reply:
Yup